### ISSUE TITLE
No server/static/data/scannet/nyuv2.json

### ISSUE THREAD

**Issue by siddhantjain** [smartscenes/sstk#2]
The wiki asks to check out server/static/data/scannet/nyuv2.json and server/static/data/scannet/nyuv2.csv for an example. I couldn't find these files in the repo. Can these be added or some other example files to follow the instructions on the wiki?

**Comment by babakhanov** [comment]
@siddhantjain +1

**Comment by siddhantjain** [comment]
@babakhanov You are right. I followed the example of the json file on that wiki page but I could not find an example for the following two files:
1. server/static/data/scannet/nyuv2.csv
2. ../../../../static/data/nyuv2/nyuv2.scans.txt

I tried writing these files based on the instructions, but I couldn't get my models to show up. So I figured if there is a complete example to follow, I can be sure that I am doing everything right.

**Comment by Coldplayplay** [comment]
Does anyone solve the ".csv" and ".json" example writing

**Comment by msavva** [comment]
@siddhantjain , @babakhanov , @Coldplayplay ,

Apologies for the long delay in responding to this issue.  We have incorporated example files and some starter data.

Please pull the latest code and refer to the instructions in https://github.com/smartscenes/sstk/wiki/Preparing-assets-for-annotation#registering-your-assets .

Let us know if you experience any issues.
